alacritty_terminal/src/term/mod.rs

```diff
@@ -954,8 +959,17 @@ impl<T> Term<T> {
             self.mode.remove(TermMode::ALTERNATE_SCROLL);
         }
         self.default_cursor_style = config.cursor.style;
+
+        self.default_title = config.window.title.clone();
         self.dynamic_title = config.dynamic_title();
 
+        if self.dynamic_title {
+            let title = self.title.as_ref().unwrap_or(&self.default_title).clone();
+            self.set_title(title);
```

Comment:
You should just send it and set to `None`, I guess. Also, you should don't only on actual change, not when `self.dynamic_title` is `true`, otherwise you'll change the current title.

Reply:
Yeah I know, that's exactly what I want to do.